sdcm/remote.py

```diff
@@ -243,7 +254,27 @@ def _ssh_ping(self, timeout=30, verbose=False):
             self.log.debug(details)
             return False
 
-    def receive_files(self, src, dst, delete_dst=False,  # pylint: disable=too-many-arguments,too-many-branches
+    def ssh_ping_thread(self):
+        while not self._ssh_up_thread_termination.isSet():
+            result = self._ssh_ping()
+            if result:
+                self._ssh_is_up.set()
+            else:
+                self._ssh_is_up.clear()
+            time.sleep(5)
+
+    def start_ssh_up_thread(self):
+        self._ssh_up_thread = threading.Thread(target=self.ssh_ping_thread)
+        self._ssh_up_thread.daemon = True
+        self._ssh_up_thread.start()
+        self._ssh_up_thread.join(5)
```

Comment:
this means it expect ssh to be up in 5sec ?

Reply:
I set it because it looks, like thread was die, after constructor has been finished.
Removed